### PR TITLE
Enabling the well-known symbols IsConcatSpreadable, ToPrimitive and HasInstance by default

### PR DESCRIPTION
**Pull request by aneeshdk**
Enabling the well-known symbols IsConcatSpreadable, ToPrimitive and HasInstance by default

### CODE REVIEW

**Comment by aneeshdk** [comment]
@akroshg @suwc @curtisman Can you please take a look?

File: lib/Runtime/Language/JavascriptConversion.cpp [hunk 1]
```diff
@@ -329,6 +329,8 @@ namespace Js
     //----------------------------------------------------------------------------
     Var JavascriptConversion::ToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * requestContext)
     {
+        Var result;
```
Comment (suwc) [inline lib/Runtime/Language/JavascriptConversion.cpp]: >Var result; [](start = 8, length = 11)

Move this into the block?

File: lib/Runtime/Language/JavascriptConversion.cpp [hunk 2]
```diff
@@ -340,6 +342,15 @@ namespace Js
         case TypeIds_Symbol:
             return aValue;
 
+        case TypeIds_VariantDate:
```
Comment (suwc) [inline lib/Runtime/Language/JavascriptConversion.cpp]: Add to description?

**Comment by suwc** [comment]
:shipit:

File: lib/Runtime/Language/JavascriptConversion.cpp (hunk 1 above)
Comment (akroshg) [inline lib/Runtime/Language/JavascriptConversion.cpp]: Also should initialize to nullptr.

---
In reply to: [86454973](https://github.com/Microsoft/ChakraCore/pull/1911#discussion_r86454973) [](ancestors = 86454973)

**Comment by curtisman** [comment]
:shipit:

**Comment by pleath** [comment]
Change to call ExecuteImplicitCall LGTM.

**Pull request merged**